src/App.js

```diff
@@ -1,25 +1,58 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState, useEffect } from 'react';
+import './index.css';
+import axios from 'axios';
+import SearchBar from './SearchBar';
+import DisplayPhotos from './DisplayPhotos';
 
 function App() {
+  const [allPhotos, setAllPhotos] = useState([]);
+  const [query, setQuery] = useState("")
+  const [error, setError] = useState(null);
+
+  const getPhotos = (e, searchquery) => {
+    e.preventDefault();
+    setQuery(searchquery);
+    if (allPhotos.length === 0) {
+      setError('No images matching your search, please try again');
+    }
+  }
+
+  // useEffect to get API data for search bar
+  useEffect(() => {
+    if (query) {
+      axios({
+        url: "https://api.unsplash.com/search/photos",
+        method: "GET",
+        dataResponse: "json",
+        params: {
+          client_id: 'TMerRFWGfjTBDdKTuLtzqQkYy-Tzm7kdNEaD67lSe-A',
+          query: query,
+          per_page: 30,
+        },
+      }).then((response) => {
+        const results = response.data.results;
+        // updating state and displaying the array images from the user's query
+        setAllPhotos(results);
+      })
+    }
+    // updating dependency array with query
+  }, [query]);
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header>
+        <h1>ünsplish</h1>
+        <SearchBar getPhotos={getPhotos} />
       </header>
+      <div className="container">
+        <DisplayPhotos photos={allPhotos} error={error} />
+      </div>
```

Comment:
Not huge but you could add a back to top button so users can scroll to the bottom and just click to get back to the search bar.